File: app/src/main/java/com/example/coffea/HomeActivity.java
package com.example.coffea;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class HomeActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_home);
    }



    public void toContributing (View view)
    {
        Intent goToContributing = new Intent (HomeActivity.this, MainContributingActivity.class);
        startActivity(goToContributing);
    }
}

File: app/src/main/java/com/example/coffea/HomeActivity.kt
package com.example.coffea

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.coffea.R
import android.content.Intent
import android.view.View
import com.example.coffea.MainContributingActivity

class HomeActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_home)


        //Hide the navigation bar
        hideNavigationBar()
    }

    //Hide navigation bar even after going out and coming back to the app
    override fun onResume() {
        super.onResume()
        hideNavigationBar()
    }

    //Hide the navigation bar
    private fun hideNavigationBar() {
        this.window.decorView.systemUiVisibility = View.SYSTEM_UI_FLAG_HIDE_NAVIGATION or
                View.SYSTEM_UI_FLAG_IMMERSIVE_STICKY
    }

    fun toContributing(view: View?) {
    //    val goToContributing = Intent(this@HomeActivity, MainContributingActivity::class.java)
    //    startActivity(goToContributing)
    }
}